ddtrace/contrib/pymongo/client.py

```diff
@@ -133,6 +137,9 @@ def run_operation(self, sock_info, operation, *args, **kwargs):
                 result = self.__wrapped__.run_operation(sock_info, operation, *args, **kwargs)
                 if result and result.address:
                     set_address_tags(span, result.address)
+                if result and self._is_query(operation):
+                    cursor = result.docs[0].get("cursor")
```

Comment:
Can the cursor be None? Can the docs array be empty? If so this can raise an exception: https://github.com/mongodb/mongo-python-driver/blob/3.5.1/pymongo/helpers.py#L142

Reply:
Just added checks for all of this, if anything is None or not as expected we abort and don't set tag